pybossa/exporter/task_csv_export.py

```diff
@@ -148,11 +149,44 @@ def _format_csv_row(self, row, headers):
     def _handle_row(self, writer, t, headers):
         writer.writerow(self._format_csv_row(self.merge_objects(t),
                                              headers=headers))
+
+    def _flatten_json_iterative(self,dictionary):
+        """Flatten objects"""
+
+        def _flatten(parent_key, parent_value):
+            """Unpack one level of nesting in json"""
+            # Unpack one level only!!!
+            
+            if isinstance(parent_value, dict):
+                for key, value in parent_value.items():
+                    new_key = u'{}{}{}'.format(parent_key, '_', key) 
+                    yield new_key, value
+            elif isinstance(parent_value, list):
+                for i, value in enumerate(parent_value, start=1): 
+                    new_key = u'{}{}{}'.format(parent_key, '_', str(i)) 
+                    yield new_key, value
+            else:
+                yield parent_key, parent_value    
+                
+        while True:
```

Comment:
can this function decorated with timeout; something similar to https://pypi.org/project/timeout-decorator/?